vignettes/ExtendingGenomicRanges.Rmd

```diff
@@ -0,0 +1,102 @@
+---
+title: "**Extending Genomic Ranges**"
+author: "Michael Lawrence, Bioconductor Team"
+date: "`r format(Sys.time(), '%d %B, %Y')`"
+package: GenomicRanges
+vignette: >
+  %\VignetteIndexEntry{Extending Genomic Ranges}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+output:
+  BiocStyle::html_document
+editor_options: 
+  markdown: 
+    wrap: 72
+---
+
+# Introduction
+
+The goal of *GenomicRanges* is to provide general containers for genomic
+data. The central class, at least from the user perspective, is , which
+formalizes the notion of ranges, while allowing for arbitrary "metadata
+columns" to be attached to it. These columns offer the same flexibility
+as the venerable and permit users to adapt *GRanges* to a wide variety
+of *adhoc* use-cases.
+
+The more we encounter a particular problem, the better we understand it.
+We eventually develop a systematic approach for solving the most
+frequently encountered problems, and every systematic approach deserves
+a systematic implementation. For example, we might want to formally
+store genetic variants, with information on alleles and read depths. The
+metadata columns, which were so useful during prototyping, are
+inappropriate for extending the formal semantics of our data structure:
+for the sake of data integrity, we need to ensure that the columns are
+always present and that they meet certain constraints.
+
+We might also find that our prototype does not scale well to the
+increased data volume that often occurs when we advance past the
+prototype stage. *GRanges* is meant mostly for prototyping and stores
+its data in memory as simple R data structures. We may require something
+more specialized when the data are large; for example, we might store
+the data as a Tabix-indexed file, or in a database.
+
+The *GenomicRanges* ppackage does not directly solve either of these
```

Comment:
Same comment as above: 

GenomicRanges is italicized as well as a hyperlink in the original vignette so we can change this to *[GenomicRanges](http://bioconductor.org/packages/release/bioc/html/GenomicRanges.html)*